src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+export interface CheckboxProps {
+	label: string;
+	id: string;
```

Comment:
id = optional

Reply:
defaults to label then?

Reply:
Geen default. Geen id meegeven = geen id op de checkbox